guides/common/modules/proc_creating-a-virt-who-configuration.adoc

```diff
@@ -0,0 +1,156 @@
+* *Filter host parents*: Comma-separated list of included cluster IDs.
+* *Exclude host parents*: Comma-separated list of excluded cluster IDs.
+For more information, see link:https://access.redhat.com/solutions/5696481[Using the "Filter Host Parents" and "Exclude Host Parents" Attributes with VMware Clusters] in the _Red{nbsp}Hat Knowledgebase_.
+endif::[]
+
+. You can configure the following logging and proxy options:
+* *Enable debugging output*: Enables debug logging for virt-who.
+* *HTTP Proxy*.
+Example: `http://_proxy.example.com_:3128`.
++
+To use no proxy, leave this field blank; this has the same result as entering `{asterisk}` in the *Ignore Proxy* field.
+* *Ignore Proxy*: Comma-separated list of host names, IP addresses, or domains to bypass existing proxy settings.
+ifdef::kubevirt-virt-who[]
+. Enter the `kubeconfig` file path.
+endif::[]
+ifdef::nutanix-virt-who[]
+. Select *Prism Central* or *Prism Element* from the *Prism Flavor* list.
+. Optional: *Enable AHV Debug*: Enables AHV internal debugging.
+This option provides additional AHV information when you enable both debugging options.
+endif::[]
+
+. Click *Submit*.
+
+[id="cli-creating-a-virt-who-configuration_{context}"]
+== CLI procedure
+
+* On {ProjectServer}, enter the `hammer virt-who-config create` command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+$ hammer virt-who-config create \
+--name _My_virt-who_Configuration_ \
+--organizations "_My_Organization_" \
+--interval 720 \ <1>
+--filtering-mode none \ <2>
+--hypervisor-id _hostname_ \ <3>
+--hypervisor-type {hypervisor-cli} \
+--hypervisor-server _{target-name}_ \ <4>
+--hypervisor-username virt_who_user \ <5>
+ifdef::vmware-virt-who,hyperv-virt-who[]
+--hypervisor-password _<password>_
+endif::[]
+--proxy '_http://proxy.example.com_:3128' \ <6>
+--satellite-url _{foreman-example-com}_
+----
+--
+<1> Optional: Specify the virtual machine information reporting interval, in minutes.
+<2> Specify `none` for no filtering of hypervisors for virt-who queries.
+Specify `whitelist` or `blacklist` to include or exclude hypervisors for virt-who queries.
+<3> Specify `hostname`, `uuid`, or `hwuuid` for the hypervisor ID format.
+* You can use `uuid` to avoid duplication if you rename a hypervisor.
++
+* You can use `hwuuid` for configurations that apply to a virtualization manager instead of an individual hypervisor.
++
+[NOTE]
+====
+You cannot change `hwuuid` to another option after virt-who starts running because this might cause duplicate entries in Subscription Manager.
+====
+
+ifdef::vmware-virt-who[]
+<4> Specify the FQDN or IP address of the vCenter Server.
+endif::[]
+ifdef::kubevirt-virt-who,openstack-virt-who,nutanix-virt-who,kvm-virt-who,hyperv-virt-who[]
+<4> Specify the FQDN or IP address of the hypervisor.
+endif::[]
+<5> Specify the name of the virt-who user you created on the hypervisor.
+<6> Optional.
+--
+
+[id="virt-who configuration example_{context}"]
+== virt-who configuration example
```

Comment:
```suggestion
.virt-who configuration example
```

Unsure about the capitalization.

Reply:
Me, too. It looks weird as lc, so I will change the capitalization.